### ISSUE TITLE
SEO audit

### ISSUE THREAD

**Issue by iandunn** [WordPress/wporg-mu-plugins#43]
Once #38 is done (especially #42), we should take a look to make sure there aren't any SEO regressions in the header or footer.

Content areas are covered by https://github.com/WordPress/wporg-news-2021/issues/143

**Comment by iandunn** [comment]
https://github.com/WordPress/wporg-news-2021/issues/143 has all the reported issues, and many are fixed, so we don't need a separate issue here anymore.
